### PR TITLE
Don't move camera during replays

### PR DESCRIPTION
**Pull request by tgjones**

### CODE REVIEW

File: src/OpenSage.Game/Logic/Orders/OrderProcessor.cs [hunk 1]
```diff
@@ -39,7 +41,11 @@ public void Process(IEnumerable<Order> orders)
                         break;
 
                     case OrderType.SetCameraPosition:
-                        _game.Scene3D.CameraController.TerrainPosition = order.Arguments[0].Value.Position;
+                        // Ignore SetCameraPosition during replays
+                        if (!_isReplay)
```
[the remark this answers is not in the document]
Comment (tgjones) [reply src/OpenSage.Game/Logic/Orders/OrderProcessor.cs]: Good point :) I’ll change it to always ignore it.

[the remark this answers is not in the document]
Comment (tgjones) [reply src/OpenSage.Game/Logic/Orders/OrderProcessor.cs]: Now `SetCameraPosition` is always ignored.

**Comment by tgjones** [comment]
@paavohuhtala does this look okay now?

**Comment by paavohuhtala** [comment]
@tgjones It does.

**Pull request merged**